frameworks/digital-outcomes-and-specialists-4/questions/brief-responses/dayRate.yml

```diff
@@ -1,6 +1,6 @@
 name: Day rate
 question: What’s the specialist’s day rate?
-question_advice: "{% if 'budgetRange' in brief %}<h2>Buyer's maximum day rate:</h2><p>{{ brief.budgetRange }}</p>{% endif %}<h2>Your maximum day rate:</h2><p>£{{ max_day_rate }}</p>"
+question_advice: '{% if "budgetRange" in brief %}<h2 class="govuk-heading-m">Buyer’s maximum day rate:</h2><p class="govuk-body">{{ brief.budgetRange }}</p>{% endif %}<h2 class="govuk-heading-m">Your maximum day rate:</h2><p class="govuk-body">£{{ max_day_rate }}</p>'
```

Comment:
Could we wrap this over multiple lines, or does that cause issues?

Reply:
No more issues than are already caused by the HTML! I'll split this out.